### ISSUE TITLE
Confidence contours instead of prediction interval contours in viz_funnel()

### ISSUE THREAD

**Issue by bwiernik** [Mkossmeier/metaviz#7]
I just made a funnel plot using viz_funnel() and was surprised that the summary effect contours looked so different from the results of `metafor::funnel()` or `psychmeta::plot_funnel()`. Looking into the code, I see that it's because metaviz plots contours for the prediction/random effects (incorporate τ), rather than the fixed effects confidence interval (just incorporating SE). I can see the value of a prediction interval (as you note in the documentation, to identify potential extreme outliers, using the "region where one expects 95% of all studies to fall"). But these contours are less useful the common use of a funnel plot to illustrate potential homogeneity vs heterogeneity (traditional confidence cc

Would it be possible to make the type of contours customizable? For example, `contours` (or `fe_contours`) could provide traditional confidence contours showing the fixed effects confidence interval, as generated by metafor (useful for evaluating homogeneity), whereas `pred_contours` could provide the current prediction interval contours (useful for evaluating potential outliers)?

If you are amenable, I can write a pull request for it. At the least, I think it would be good to make it clearer in the documentation that the contours incorporate τ and so they don't align with methods used by other funnel plot functions/programs.

**Comment by Mkossmeier** [comment]
Thank you very much for your valueable feedback!

I just updated the viz_funnel function accordingly. Function viz_funnel now has an additional argument "contours_pred", which can be set to either TRUE or FALSE (default is FALSE for consistency with results from other packages as you mentioned) and which controls if τ is incorporated in computing the contours around the summary effect. I also updated the documentation accordingly and hope it is now more clear in this regard.

This new version is not yet on CRAN but you can try it out by installing it from github if you like ( devtools::install_github("Mkossmeier/metaviz"))

Best, 
Michael

**Comment by bwiernik** [comment]
Thinking about it, the name of the argument might better be labeled as `contours_fe`. The previous behavior is a confidence interval, just one that incorporates the random effects variance into it. 

cf. https://psycnet.apa.org/record/2019-07158-001

**Comment by bwiernik** [comment]
This change also could let you change the default `method` argument of `rma` objects to whatever method the `rma` object used. I can make a PR that will do that elegantly using S3.

**Comment by Mkossmeier** [comment]
Giving it some tought, I changed the parameter label to `contours_type`, which can now be either "FEM" (classic contours) or "REM" (contours also incorporating the between study heterogenity τ^2).

I also agree that it is convenient to keep and extract the method argument from the `rma` object when using `rma` objects from package metafor as input of metaviz. I changed the behavior  of metaviz accordingly in the latest commit.

Thanks again for your valueable feedback!
